### DISCUSSION TITLE
[Bug / Feature] Missing template auto-complete in Routes

### DISCUSSION THREAD

**Discussion by OwenMelbz** [craftcms/cms#8984] [Ideas]
### Description

Feel free to close if this is expected behaviour, however I noticed when creating a route the "template" field doesn't have the template auto-complete that the other areas of the CMS has.

![image](https://user-images.githubusercontent.com/1094740/103407287-0f3f2800-4b56-11eb-8fb1-7f10532bb372.png)

Not sure if this is a bug or what, so feel free to categorise as you feel appropriate!

### Steps to reproduce

1. Settings -> Routes -> New route
2. Use the "template" field

### Additional info

- Craft version: 3.5
- PHP version: 7.4
- Database driver & version: Mysql 8
- Plugins & versions: N/A

**Comment by brandonkelly** [comment]
Currently expected as those modals are created by JS, and we don’t have a way to instantiate those autosuggest fields from JS yet.
